src/refactorings/convert-switch-to-if-else/convert-switch-to-if-else.ts

```diff
@@ -34,7 +38,7 @@ function createVisitor(
   selection: Selection,
   onMatch: (
     path: t.NodePath<t.SwitchStatement>,
-    convertedNode: t.IfStatement | t.SwitchStatement
+    convertedNode: t.Statement
```

Comment:
This could also be
```suggestion
    convertedNode: t.IfStatement | t.SwitchStatement | t.Statement[]
```
but maybe no

Reply:
It's OK to return `t.Statement` here I think.